src/main/java/hudson/remoting/ClassFilter.java

```diff
@@ -86,7 +86,32 @@ public final Class check(Class c) {
      * A set of sensible default filtering rules to apply,
      * unless the context guarantees the trust between two channels.
      */
-    public static final ClassFilter DEFAULT = createDefaultInstance();
+    public static final ClassFilter DEFAULT;
+
+    static {
+        try {
+            DEFAULT = createDefaultInstance();
+        } catch (ClassFilterException ex) {
+            LOGGER.log(Level.SEVERE, "Default class filter cannot be initialized. Remoting will not start", ex);
+            throw new ExceptionInInitializerError(ex);
+        }
+    }
+
+    /**
+     * Adds an additional exclusion to {@link #DEFAULT}.
+     * 
+     * Does nothing if the default list has already been customized via {@link #FILE_OVERRIDE_LOCATION_PROPERTY}.
+     * This API is not supposed to be used anywhere outside Jenkins core, calls for other sources may be rejected later.
```

Comment:
I suppose it would be legitimate to call from a plugin if we found that there was a dangerous class in a library bundled with a plugin (but not in Jenkins core).